django/db/backends/sqlite3/introspection.py

```diff
@@ -9,7 +9,7 @@
 from django.db.models import Index
 from django.utils.regex_helper import _lazy_re_compile
 
-FieldInfo = namedtuple('FieldInfo', BaseFieldInfo._fields + ('pk', 'has_json_constraint'))
+FieldInfo = namedtuple('FieldInfo', BaseFieldInfo._fields + ('pk', 'has_json_constraint', 'db_comment'))
```

Comment:
Why you added `db_comment` if it's not supported? :thinking:

Reply:
If it doesn't exists, `sqlite_connection.introspection.get_table_description(cursor, table_name)`  raise AttributeError() or "FieldInfo () args expected 12 but 13..."